.circleci/config.yml

```diff
@@ -24,6 +24,6 @@ jobs:
 
       - run: yarn lint
 
-      - run: yarn build # we need to do this to prebuild some babel stuff
+      - run: GATSBY_CPU_COUNT=1 yarn build
```

Comment:
This is a fix for:

https://circleci.com/gh/storybookjs/frontpage/586?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

Not sure why this PR triggered this to start happening -- maybe because I added a few more images (perhaps the PNG image).